plugins/foreman_openscap/0.5/index.md

```diff
@@ -80,9 +80,15 @@ Foreman OpenSCAP (foreman_openscap), Smart Proxy OpenSCAP (smart_proxy_openscap)
 
 ### 2.1 Installing foreman_openscap
 
-On the Foreman server:
+The easiest and recommended way is using foreman-installer. It's as easy as running this command
+
+	> foreman-installer --enable-foreman-plugin-openscap
+
+If you prefer manual way, you can install it from package like this:
 
 	> yum install tfm-rubygem-foreman_openscap
+	> foreman-rake db:migrate
+	> foreman-rake db:seed
```

Comment:
Is the required when using RPMs? In the DEBs we do have these tasks when required.